extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/HibernateOrmRuntimeConfigPersistenceUnit.java

```diff
@@ -107,6 +107,12 @@ public boolean isAnyPropertySet() {
         @ConfigItem(name = ConfigItem.PARENT, defaultValue = "none")
         public String generation = "none";
 
+        /**
+         * Specifies the statement delimiter to be used for the generated DDL scripts.
+         */
+        @ConfigItem(defaultValue = ";")
+        public String delimiter;
```

Comment:
Assuming we would want to have the value default to some suitable value for the dialect (DB kind) at hand, how would we specify that? One option would be to declare the field as `Optional<String>` and without a `defaultValue` and instead with a corresponding `defaultValueDocumentation` to document this. Or is there another pattern?

I still don't know what default value to use for the various DB kinds. I *think* `;` should be fine for all currently supported DBs.